web/src/components/OverlayFooter.tsx

```diff
@@ -8,30 +9,21 @@ type OverlayFooterProps = {
 
 const OverlayFooterContainer = styled(Paper)`
   display: flex;
-  white-space: nowrap;
   padding: ${props => props.theme.spacing(2)};
-  background-color: rgb(255 255 255 / 50%);
+  background-color: ${props => props.theme.palette.surface.main}80;
+  border-radius: 0;
 
   ${props => props.theme.breakpoints.up('md')} {
     padding: 0 ${props => props.theme.spacing(1)};
   }
```

Comment:
🤔 Since there is no overlay footer on screen sizes smaller then `md`, this condition is unnecessary.
🙃 I think it looks better without the horizontal 8px padding which would also give us more space still :)
```suggestion
  padding: 0;
```